examples/seismic/tti/wavesolver.py

```diff
@@ -10,26 +10,31 @@ class AnisotropicWaveSolver(object):
     and encapsulates the time and space discretization for a given problem
     setup.
 
-    :param model: Physical model with domain parameters
-    :param source: Sparse point symbol providing the injected wave
-    :param receiver: Sparse point symbol describing an array of receivers
-    :param time_order: Order of the time-stepping scheme (default: 2)
-    :param space_order: Order of the spatial stencil discretisation (default: 4)
+    Parameters
+    ----------
+    model : Model
+        Physical model with domain parameters
+    geometry : AcquisitionGeometry
+        Source and receivers geometry
+    space_order : Int
+        Order of the spatial stencil discretisation (default: 2)
 
-    Note: space_order must always be greater than time_order
+    Note: space_order must always be an even number
```

Comment:
1) numpydoc for notes:
```
Notes
-----
...
```

2) raise a ValueError if that's not honored

Reply:
Done